frontend/src/metabase/visualizations/components/LineAreaBarChart.jsx

```diff
@@ -35,7 +35,7 @@ const isDimensionDimensionMetric = (cols, strict = true) =>
     isMetric(cols[2])
 
 const isDimensionMetricMetric = (cols, strict = true) =>
```

Comment:
this looks right to me.  i mean, for this to qualify it should have 3 or more columns